documentation/book/con-customizing-pod-disruption-budgets.adoc

```diff
@@ -0,0 +1,47 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='con-customizing-pod-disruption-budgets-{context}']
+= Customizing Pod Disruption Budgets
+
+{ProductName} automatically creates a Pod Disruption Budget for every `StatefulSet` or `Deployment` which it creates.
+By default, the Pod Disruption Budgets always allow up to one Pod to be unavailable at a time by setting the `maxUnavailable` value in `PodDisruptionBudget.spec` to `1`.
+You can change this default setting using the Pod Disruption Budget template.
+
+In the Pod Disruption Budget template you can - in addition to Labels and Annotations - customize the `maxUnavailable` field as well:
+
+[table,stripes=none]
```

Comment:
I have several problems with this table. 

* It has a subset of the properties of the one in the API reference, but it's not clear that the properties in this table is not exhaustive.
* It duplicates the documentation of `maxUnavailable` using different words. This is a pattern we're already using elsewhere, but it means there's content which needs maintaining in two places, which is a maintenance headache in the long run.

If we're really going to stick with this pattern of having concepts duplicating the content of the API reference then I think we should do it in a more intelligent way. For example, we could generate a `fragment-PodDisruptionBudgetTemplate.adoc` file with this table in it. That would allow you to write useful verbiage around the configurable properties without needing to duplicate the tables.